client/src/components/layout/Header.js

```diff
@@ -2,34 +2,33 @@ import React from "react"
 import { Navbar, Nav, Image } from "react-bootstrap"
 import { Link } from 'react-router-dom'
 import {useSelector} from 'react-redux';
+import {BsPeopleCircle} from 'react-icons/bs'
 
 import "./Header.css"
 
-import profilePicture from "../../images/user.png"
+import { Fragment } from "react";
 
 /**
  * This function provides header for Layout.js
  * @returns Header component to be rendered in Layout.js
  */
-function Header() {
+const Header = ({title, urls}) => {
     const userName = useSelector(state => state.user?.givenName + " " + state.user?.familyName);
     //TODO: Remove hard-coded team id and publications id from the links
     return (
-        <Navbar bg="primary" variant="dark" className="header">
-            <Navbar.Brand><Link className="header-brand" to="/dashboard">Researchify</Link></Navbar.Brand>
-            <Nav className="mr-auto">
-                <Nav.Link><Link className="header-link" to="/dashboard/profile">Profile</Link></Nav.Link>
-                <Nav.Link><Link className="header-link" to="/publications/team/606bb59c22201f529db920c9">Publications </Link></Nav.Link>
-
-            </Nav>
-            <Nav>
-                <Nav.Link className="header-username">
-                    <Link className="header-link" to="/dashboard/profile">
-                        <Image className="profile-picture" src={profilePicture} roundedCircle /> {userName}
-                    </Link>
-                </Nav.Link>
-            </Nav>
-        </Navbar>
+        <Fragment>
+            <Navbar className="header" sticky="top">
+                <Navbar.Brand><Link className="header-brand" to={urls.dashboard}>{title}</Link></Navbar.Brand>
+                <Nav className="mr-auto" />
+                <Nav>
+                    <Nav.Link className="header-profile">
+                        <Link className="header-link" to="/dashboard/profile">
+                            <BsPeopleCircle className="header-profile-icon" /> {userName}
```

Comment:
Just wondering, is the username supposed to show next to the profile icon in the top right? It might be good to hardcode a name for demo purposes

Reply:
How about the profile editing page? Do we also put a hardcoded value in profile edit page for now? XD

Reply:
Yeah, I suppose so for demo purposes, like pretend we have a default user logged in